buildpacks/builder.go

```diff
@@ -0,0 +1,67 @@
+package buildpacks
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/logging"
+	"io"
+	"os"
+)
+
+type Builder struct {
+	Verbose   bool
+	registry  string
+	namespace string
+}
+
+func NewBuilder(registry, namespace string) *Builder {
+	return &Builder{registry: registry, namespace: namespace}
+}
+
+var lang2pack = map[string]string{
+	"java": "quay.io/mvasek/faas-quarkus-builder",
```

Comment:
Do you want to upload these to the boson project at Quay?

Reply:
Probably that's better palce.

Reply:
@matejvasek looking forward to a PR of that buildpack :smiley:

Reply:
@lance  https://github.com/boson-project/buildpacks/pull/1

Reply:
Hahah - I shold have looked first :)